src/rendering/framebufferrenderer.cpp

```diff
@@ -252,30 +295,6 @@ void FramebufferRenderer::initialize() {
         LERROR("Exit framebuffer is not complete");
     }
 
-    //===================================//
```

Comment:
removed because it wasn't really used? 

I remember some discussion about this at some point...